### ISSUE TITLE
ignore this

### ISSUE THREAD

**Issue by vesper8** [carlosflorencio/laracasts-downloader#41]
nevermind.. I am an idiot. I had copied .env.example to .env but had only put the right username/password in the .env.example =/

However.. this did detect one bug.. it shows

```
====================================
Authenticating
====================================
> Successfull!
```

even with wrong info.. in this case the defaults
EMAIL=test.com
PASSWORD=password

**Comment by vesper8** [comment]
nvm

**Comment by carlosflorencio** [comment]
I'll investigate tonight.

**Comment by carlosflorencio** [comment]
Fixed in 5a764c7e4abbd47ed0f2e4d425422b0ab674fcee
